src/controllers/userContent.js

```diff
@@ -11,7 +11,7 @@ exports.get = async (req, res) => {
       const user = await getUser(username);
       const likedContent = getUserContent(user.like);
       res.render('usercontent', {
-        likedContent, usercontent: true, username, initial,
+        likedContent, usercontent: true, backLink: '/login', username, initial,
```

Comment:
I purposefully removed the backlink from the /usercontent page as the logged in user should not be taken to the log in page on back button, there should be only profile page to go to